src/metric.ts

```diff
@@ -0,0 +1,9 @@
+export default class Metric {
+  name: string;
+  duration: number;
+
+  constructor(metricName: string, duration: number) {
```

Comment:
You can make it simpler and use `name` instead of `metricName`

Reply:
Done.